docs/authorization.md

```diff
@@ -4,7 +4,7 @@ With `gcloud-node` it's incredibly easy to get authorized and start using Google
 
 If you are running this client on Google Compute Engine, we handle authorization for you with no configuration. You just need to make sure that when you [set up the GCE instance][gce-how-to], you add the correct scopes for the APIs you want to access.
 
-```js hljs-class
+```hljs-class
```

Comment:
Bug 1: We don't need the `js`. `hljs-class` is enough to be picked up by our docs site and appropriately syntax highlighted. The only loss is no syntax highlighting on the GH viewer: https://github.com/stephenplusplus/gcloud-node/blob/b21744fc3ef07eb8a260af7ecd4570c611420d9d/docs/authorization.md but I think we want our docs site to look the best.